partcad/src/partcad/actions/part_actions.py

```
@@ -0,0 +1,142 @@
+    :param source_path: Path to the source file
+    :param config: Additional configuration for the part
+    """
+    config = config or {}
+    source_path = Path(source_path)  # Ensure source_path is a Path object
+    target_path = Path(project.path) / f"{name}.{kind}"  # Define the target file path
+
+    pc_logging.info(f"Importing the part {name} of type {kind} from {source_path} to {target_path}")
+
+    # Check if the source file exists
+    if not source_path.exists():
+        raise ValueError(f"Source file '{source_path}' does not exist.")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)